File: Software-Technologies/IMDB/C# Skeleton/IMDB/Models/Film.cs
namespace IMDB.Models
{
    using System.ComponentModel.DataAnnotations;

    public class Film
    {
        [Key]
        public int Id { get; set; }

        public string Name { get; set; }

        public string Genre { get; set; }

        public string Director { get; set; }

        public int Year { get; set; }




    }
}

File: Software-Technologies/IMDB/JavaScript Skeleton/models/film.js
const Sequelize = require('sequelize');

module.exports = function(sequelize){
    return sequelize.define('Film',{
        name: {
            type: Sequelize.TEXT
        },
        genre: {
            type: Sequelize.TEXT
        },
        director: {
            type: Sequelize.TEXT
        },
        year: {
            type: Sequelize.INTEGER
        }
    },{
        timestamps: false,
    })
};